rm-services-rh-stub/app.py

```diff
@@ -0,0 +1,101 @@
+from flask import Flask
+from flask import Response
+from sample_stub import Sample_Stub
+from collex_stub import Collex_Stub
+from collex_instr_stub import Collex_Instr_Stub
+from case_stub import Case_Stub
+from iac_stub import Iac_Stub
+
+import logging
+app = Flask(__name__)
+
+#override the default logging configuration so that info messages get output
+
+logging.basicConfig(level=logging.INFO)
+
+
+@app.route('/')
+def hello():
+    return "Hello from the RM services stub for RH"
+
+""" Route handler to deal with getting sample attributes from the sample service. 
+The sampleid could be any sampleid that is passed in """
+
+@app.route('/samples/<sampleid>/attributes')
+def get_sample_attributes(sampleid):
+    my_sample_stub = Sample_Stub(sampleid)
```

Comment:
In order for the function comments to be docstrings, which are returned by calling help() on the function, they need to be put at the start of the function. I suggest doing this for other functions too.

```
    """ Route handler to deal with getting sample attributes from the sample service. 
    The sampleid could be any sampleid that is passed in """
    my_sample_stub = Sample_Stub(sampleid)
```